spec/fixtures/18ZOO/or_power.a_spoonful_of_sugar.not_yet_working

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
I don't think we need these "not yet working" fixtures checked in.

Reply:
ok, I'll remove, thanks